ambry-clustermap/src/main/java/com.github.ambry.clustermap/CompositeClusterManager.java

```diff
@@ -213,6 +219,25 @@ public void close() {
       helixClusterManager.close();
     }
   }
+
+  /**
+   * Check for partition list equivalency
+   * @param partitionListOne {@link List} of {@link PartitionId}s to compare
+   * @param partitionListTwo {@link List} of {@link AmbryPartition}s to compare
+   * @return {@code true} if both list are equal, {@code false} otherwise
+   */
+  private boolean checkPartitionListEquivalency(List<PartitionId> partitionListOne,
```

Comment:
Call this `areEqual()`? Better readability in general for boolean checks for eg.: ```if (!areEqual(partitionListA, partitionListB))```